dask/layers.py

```diff
@@ -345,20 +345,7 @@ def fractional_slice(task, axes):
 class DataFrameLayer(Layer):
```

Comment:
Should this class even exist anymore?

Reply:
At this point, it acts only as a "label" that we **could** use for complex HLG optimizations (but that we are not actually using at all yet). So, we could certainly remove it, but will need to add something like it for multi-layer column projection and/or multi-layer predicate pushdown.